Finder.php

```diff
@@ -134,7 +134,20 @@ public function in($path)
         }
 
         foreach ($path as $p) {
-            $this->_paths[] = $p;
+            if (1 === preg_match('/[\*\?\[\]]/', $p)) {
+                $iterator = new Iterator\CallbackFilter(
+                    new Iterator\Glob(rtrim($p, DS)),
+                    function ($current) {
+                        return $current->isDir();
+                    }
+                );
+
+                foreach ($iterator as $path => $_fileInfo) {
```

Comment:
Can you change the variable name here, because `$path` is the same as the upper `foreach` value variable (line 136).